integration-tests/runtime-tests/test/globals.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Please do not add `.js` file as they are generated from their typescript definition